tests/html/load-empty.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <title>load event Test for empty resources</title>
+    <script src="../../src/html-imports.js"></script>
+    <script>
+      WCT = {
+        waitFor: function(callback) {
+          HTMLImports.whenReady(callback);
+        }
+      };
+    </script>
+    <script src="../../../web-component-tester/browser.js"></script>
+    <script>
+      window.loadEvents = 0;
+      (function() {
+        function importLoaded(event) {
+          window.loadEvents++;
+          chai.assert.equal(event.type, 'load', 'event type should be `load`');
+          chai.assert.ok(event.target.import, 'link should have `import` defined');
+          // https://github.com/webcomponents/html-imports/issues/3
+          // chai.assert.ok(event.target.import instanceof Document, 'load event target can be used to find imported document');
+        }
+
+        window.importLoaded = importLoaded;
+      })();
+    </script>
+    <link rel="import" href="imports/load-empty.html" onload="importLoaded(event)">
+
+  </head>
+  <body>
+    <script>
+      test('load empty resource', function() {
+        chai.assert.equal(loadEvents, 1, 'expected # of load events');
+      });
+      test('load empty resource asynchronously', function(done) {
+        loadEvents = 0;
+        setTimeout(function(){
+          var anotherHTMLImport = document.createElement('link');
+          anotherHTMLImport.rel = 'import';
+          anotherHTMLImport.href = 'imports/load-empty-async.html';
+          anotherHTMLImport.onload = importLoaded;
+          document.head.appendChild(anotherHTMLImport);
+
+          setTimeout(function(){
```

Comment:
Yes, it would make the tests dependent between them because you'd get the cumulated total # of imports.
I was a bit skeptical for the `setTimeout` of 100ms which is a bit random value, we could just add a listener on load and call `done()` in the callback, but that's a nit.
Let's keep these tests units 👌 This LGTM